app/routes.py

```diff
@@ -1,4 +1,14 @@
-from flask import render_template, url_for, flash, redirect, request, Blueprint, session, Response
+from flask import (
```

Comment:
Since this is just formatting, can we squash it with another change to this file?